ui/src/app/component/search/search.component.html

```diff
@@ -114,6 +114,9 @@ <h4>No results&hellip;</h4>
               <th>Username</th>
               <th class="text-truncate">Staff Code</th>
               <th>Team(s)</th>
+              <th class="text-right " >
+                <span class="oi oi-data-transfer-download" style="cursor: pointer;" (click)="exportSearchResultToCSV()" title="Export to CSV..."></span >
+              </th>
               <th *ngIf="searchParams.includeInactiveUsers" class="text-nowrap">End Date</th>
               <th></th>
```

Comment:
I think this will misalign the End Date header when doing a search with inactive users. Would this be better?
```suggestion
              <th *ngIf="searchParams.includeInactiveUsers" class="text-nowrap">End Date</th>
              <th class="text-right">
                <span class="oi oi-data-transfer-download" style="cursor: pointer" (click)="exportSearchResultToCSV()" title="Export to CSV..."></span>
              </th>
```
This makes use of the existing empty header cell and also right-aligns the icon